exonum-java-binding-time-service/src/test/java/com/exonum/binding/time/TimeSchemaProxyIntegrationTest.java

```diff
@@ -29,8 +29,9 @@
 import java.util.function.Consumer;
 import org.junit.jupiter.api.Test;
 
+//TODO: fix after native part implementation
```

Comment:
Please `@Disable/Ignore` it, leaving the TODO, so that this PR builds and can be integrated without native part.